### DISCUSSION TITLE
Sonoff SNZB-06P - Problemi: non mostra occupazione

### DISCUSSION THREAD

**Discussion by Medusa320** [indomus/forum#3114] [HUB: Home Assistant]
Buongiorno a tutti,
ho un problema con Sonoff SNZB-06P: l'ho aggiunto senza problemi ad Home Assistant tramite Zigbee2MQTT, ma quando vado nelle impostazioni vedo solo il moviemento; come posso abilitare anche l'occupazione? 
Anche tramite Zigbee2MQTT non vedo altri valori.
Ho provato anche ad aggiungere un gateway a Smarthome e ad aggiungere il Sonoff al gateway, ma viene riconosciuto solo come PIR Sensor. Qualcun'altro ha lo stesso problema?
Grazie a tutti per l'aiuto
![immagine](https://github.com/indomus/forum/assets/120176125/a30089cc-55fa-48d9-b9b1-684fd85542e2)
![immagine](https://github.com/indomus/forum/assets/120176125/795b489a-ce96-44e0-92d1-a230f9d98830)
![immagine](https://github.com/indomus/forum/assets/120176125/9a412065-aa09-41cb-a448-a61ca311919e)

**Comment by Scialla** [comment]
Ma scusa, dove lo vedi il movimento?
Ti espone il sensore `occupancy` che è il sensore di presenza; Non è un PIR espone solo la presenza, poi che nel frontend te la chiami "motion" non vuol dire nulla, ma è un sensore di presenza e infatti, su z2m, dice correttamente `occupancy`

Poi ci sono alcuni sensori di presenza che espongono altre entità, ad esempio la modalità di presenza, statica o in movimento piccolo o grande, la distanza etc, ma questo espone solo quelli
https://www.zigbee2mqtt.io/devices/SNZB-06P.html

**Reply by Medusa320** [reply]
Ciao, grazie per la risposta
Forse ho un po di confusione, ma andiamo con ordine; ho deciso di provare il sensore dopo la [recensione su in domus](https://indomus.it/componenti/itead-sonoff-snzb-06p/) e li nel capitolo HomeAssistant vengono presentati Movimento e Occupazione
![image](https://github.com/indomus/forum/assets/120176125/85e09f93-d7ad-469b-ad4f-41cfcd6843ac)
Realizzo solo adesso che lo screenshot é stato fatto con ZHA.

Qualche test di presenza l'ho fatto, ma mi sembrava che rilevasse solo il movimento e non la presenza statica; aggiungendo il fatto che aggiunto a SmartHome tramite gateway si presentasse com "PIR Sensor", mi ha fatto temere che fosse una versione farlocca del sensore.

Proverò appena ho tempo a fare qualche ulteriore test di occupazione, ed eventualmente aggiungere ZHA
Grazie per gli spunti

**Reply by Scialla** [reply]
Tieni presente che i sensori di presenza vanno tarati al meglio e, almeno a me, se rimango immobile per diversi secondi, non vengo più rilevato... poi basta aprire la bocca e ti vede...
